src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -52,47 +52,47 @@ const CreateConceptForm = props => (
           <div className="form-group col-md-5">
             <label htmlFor="class">Class</label>
             <div>
-              { (props.concept.toString().trim() === 'symptom-finding') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control symptom-finding"
-                  required
```

Comment:
Can you explain how this change is related to the ticket?

Reply:
All this are formatting changes that I was reverting. They are not part of this ticket.